test/functional/plugin/lsp_spec.lua

```diff
@@ -1043,6 +1043,72 @@ describe('LSP', function()
       ]])
     end)
   end)
+  describe('lsp.util.locations_to_items', function()
+    it('Convert Location[] to items', function()
+      local expected = {
+        {
+          filename = '',
+          lnum = 1,
+          col = 3,
+          text = 'testing'
+        },
+      }
+      local actual = exec_lua [[
+        local buffer = vim.api.nvim_create_buf(false, true)
+        vim.api.nvim_buf_set_name(buffer, 'dummy')
+        vim.api.nvim_buf_set_lines(buffer, 0, -1, false, {
+          "testing";
+          "123";
+        })
+        local locations = {
+          {
+            uri = vim.uri_from_bufnr(buffer),
+            range = {
+              start = { line = 0, character = 2 },
+              ['end'] = { line = 0, character = 3 },
+            }
+          },
+        }
+        return vim.lsp.util.locations_to_items(locations)
+      ]]
+      actual[1].filename = ''
```

Comment:
I think this part is kind of weird. Can we load a file using something like we do elsewhere for `file://fake/uri` or something like that? I don't remember exactly what that was though... I can look if you can't find it.

Reply:
Yep it was weird. Thank you for the pointer to `fake/uri` - I changed it now.